docs/model_training-ngboost.ipynb

```diff
@@ -0,0 +1,639 @@
+    "result_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While this training dataset is far too small to draw real conclusions, these metrics can give you an idea of how well the model performed on a holdout test set (20% of the training data as we specificed by the `test_size` parameter in the configuration. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can write the model to a json file that can be loaded later:\n",
+    "\n",
+    "```python\n",
+    "test_vehicle.to_file(\"Test_Vehicle.bin\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_vehicle.to_file(\"Test_Vehicle.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To retrieve a saved model from a json file for further use can be done by doing the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_vehicle = pt.Model.from_file(\"Test_Vehicle.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## RouteE Compass Integration\n",
+    "\n",
+    "If you want to use this model with RouteE Compass, you can export any of the estimators as binary file and that can be loaded into RouteE Compass.\n",
+    "\n",
+    "In this case, we have three estimators:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'speed_mph': <nrel.routee.powertrain.estimators.ngboost_estimator.NGBoostEstimator at 0x1597b3910>,\n",
+       " 'grade_dec&speed_mph': <nrel.routee.powertrain.estimators.ngboost_estimator.NGBoostEstimator at 0x159802b90>,\n",
+       " 'grade_dec&road_class&speed_mph': <nrel.routee.powertrain.estimators.ngboost_estimator.NGBoostEstimator at 0x159890850>}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "test_vehicle.estimators"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this example, we'll take the estimator with speed and grade as features and export it to a binary file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_vehicle.estimators['grade_dec&speed_mph'].to_file(\"test_vehicle_speed_grade.bin\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can load `test_vehicle_speed_grade.bin` into RouteE Comapss"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "routee_compt",
```

Comment:
Can you rerun this notebook with just the default `Python 3 (ipykernel)` kernel? We need this when using jupyter-book to build the documentation.

Eventually, I think we'll switch over to putting these examples in python scripts and then converting them to notebooks at the time we build the documentation (which will eliminate this problem).